charter/index.md

```diff
@@ -0,0 +1,228 @@
+---
+title: Font and Text Community Group Charter
+---
+This charter:  {{ page.url | remove: "index.html" | prepend: site.url }}
+
+Feedback about this document:
+    https://github.com/w3c/font-text-cg/issues
+
+## Goals
+
+The Font and Text community group gathers individuals and organizations
+interested in discussing and developing specifications and implementations
+for technologies
+which operate on and at the interface between
+text encoding and font formats.
+Examples of such technologies are
+shaping engines and text layout applications.
+
+## Scope of work
+
+Under the present charter, the Community Group will
+determine a structure and process
+under which such specifications may be created
+by which the greatest number of interested parties may be able to participate.
+Until such a structure and process is determined,
+no technical development of specifications or implementations
+will be undertaken by this Community Group
+and no technical contributions will be accepted.
+
+## Deliverables
+
+No technical specifications will be produced by this Community Group
+and no technical contributions are
+to be solicited
+or accepted.
+
+* This Community Group aims to produce a proposed Charter,
+to be used to set up a technical group,
+possibly a W3C Working Group,
+under which members of this community should conduct technical work.
+
+* If, after producing such a charter,
+the Community Group determines that its current composition is adequate
+for conducting its work,
+then the charter produced is
+to be adopted as an updated Charter for this CG.
+
+* If the Community Group determines that it would be appropriate
+to divide the technical work into several venues,
+it may produce several such proposed charters
+rather than a single one.
+
+* Various supporting documents and articles
+describing the problem space
+may also be produced.
+
+All deliverables of this Community Group will use the
+[W3C Software and Document License](http://www.w3.org/Consortium/Legal/2015/copyright-software-and-document).
+
+This group maintains an [index](https://w3c.github.io/font-text-cg/work-items/)
+of the documents it is working on.
+
+## Community and Business Group Process and Patent Policy
```

Comment:
Dropped.

Reply:
I think we need to consider the possibility that the work of the CG will eventually be reorganized as WG, in which case strict adherence to W3C Patent Policy would be a requirement. If we say nothing about it now, we might find ourselves in a position where things contributed to the CG (without any declared patent policy) could be excluded when/if the WG is formed.

Reply:
Okay, but the *section* says nothing specific about patent policy anyway. It does explicitly state that everything done as part of the CG is done under the W3C CLA, which does spell out patent policy. In other words it's impossible to contribute without that being the declared patent policy already. Are you objecting to this draft being ratified without an extra section on this? Or to the title of this section being changed? Or neither?

Reply:
Neither, my comment wasn't an objection - it was an attempt to make sure that we don't overlook something important that might come back and bite us later. For as long as there is clarity that all contributions are covered by CLA, and there is a clear boundary setup between individual participants who represent their organizations vs. those who are not - we are technically ok. But it does put an additional burden on chair(s) to keep track of it all, because, if in the heat of the discussion someone who does not have organizational commitment makes a good proposal and its accepted as generally good idea - it would be the chair who has to remember that this proposal can't be accepted before the request to make a contribution is made, _and_ the organizations commitment is in place. We do have it mentioned in the "Process" section, but I find this kind of arrangement to be burdensome for the chair of the CG.

Reply:
>  it would be the chair who has to remember that this proposal can't be accepted before the request to make a contribution is made

@xfq @r12a @svgeesus Can the IPR checking bot that is available for working groups be made available to CGs to automate this part?

For now, there's no urgency, since the group isn't taking on any technical work yet, but once that happens, automating this kind of check would be nice.

Reply:
I just filed https://github.com/w3c/ash-nazg/issues/186